org.civicrm.reqangular/src/common/services/api.js

```diff
@@ -42,6 +42,20 @@ define([
       return JSON.stringify(_.merge(defaults, params || {}));
     }
 
+    /**
+     * Constructs a GET query string exactly the same way as $http service does
+     *
+     * @param  {Object} json that is passed as "params" to $http
+     * @return {String} query string
+     */
+    function jsonToQueryString (json) {
+      return '?' + Object.keys(json).sort().map(function (key) {
```

Comment:
if you really need this function (and again please triple check, i have the feeling we are reinventing the wheel here), please move it toward the bottom, follow the style guide